src/main/java/com/jcabi/http/mock/MkAnswer.java

```diff
@@ -52,6 +52,7 @@
  * @since 0.10
  */
 @Immutable
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@mkordas I'm thinking about if it would be better to eliminate the other method right away. It's not used anymore (or should not be used anymore, because it can't handle binary correctly).